editoast/src/schema/track_section.rs

```diff
@@ -63,6 +68,21 @@ impl OSRDIdentified for TrackSection {
     }
 }
 
+impl TrackSection {
+    fn bbox(geom: &geos::geojson::Geometry) -> BoundingBox {
+        BoundingBox::from_geometry(geom.clone())
+            .expect("track sections can only be represented by LineStrings")
+    }
```

Comment:
I'll add the check at building time but I prefer to leave the message here because it is a hint on what could go wrong, besides the TrackSection could have been mutated after initialization.

Reply:
There is no TrackSection::new as it is either initialized by default() or via deserialization. Checking the geometry would require a custom deserializer but I don't think its worth the work since everything else breaks anyway should a track section not be a linestring :D